challenge_5/java/jdfurlan/src/FindTheDifference.java

```diff
@@ -0,0 +1,21 @@
+
+public class FindTheDifference {
+
+	public static void main(String[] args) {
+		String s = "abcd";
+		String t = "dbeca";
+
+		boolean[] b = new boolean[128];
```

Comment:
You're right! I added a fix though it is a bit hacky. In the event that all chars are the same, and given the fact that we know one char HAS to be added to the original string, we can just print any char in the string if we never found a unique char.